data/data-pipeline/data_pipeline/etl/sources/census/etl_utils.py

```diff
@@ -107,7 +107,7 @@ def check_census_data_source(
         # check if census data is found locally
         if not os.path.isfile(census_data_path / "geojson" / "us.json"):
             logger.info(
-                "No local census data found. Please use '-d aws` to fetch from AWS"
+                "No local census data found. Please use '-s aws` to fetch from AWS"
```

Comment:
What does -s and -d do? Is this the same as the AWS CLI commands we run in the github action yml files?

Reply:
-s specifies the data source for the census data (either aws or local). it was -d for a minute but that's confusing with a flag used elsewhere, which specifies a source data set  for indicator data (eg ejscreen or hud)